desktop/apps/auction_support/client/bid_form.coffee

```diff
@@ -40,6 +40,10 @@ module.exports = class BidForm extends ErrorHandlingForm
     val = @$('.max-bid').val()
     @saleArtwork.cleanBidAmount val if val
 
+  removeRegistrationFormAndBidImmediately: =>
+    @$('.bid-and-register-form').hide()
```

Comment:
I think if this is only called from the one place, we can just combine the methods and add a comment for context.

The other question is whether this is ever called when a `'bid-and-register-form'` is not present. Do we need to be defensive at all?